GoodDollar/GoodDAPP
# Celo wallet web3provider

# Description

Replacing PrivateKeyProvider with a custom JsonRpcProvider for compatibility with android
About # (link your issue here)

- [x] - Add custom JsonRpcWithSigner class
- [ ] - Test implementation and confirm it is STILL working on web
- [ ] - Test on android, should now work as well

# How to test?
1. yarn web (or yarn run android)
2. go to localhost
3. login wallet
4. click on bridge icon (blue one in the middle of balance overview)
5. fill in amount > 1000G$
6. click next
7. click confirm

Should send user back to home-screen, and a new bridge tx should be added to feedlist
If anything fails, there are likely rpc call errors in dev-tools visible

# How Has This Been Tested?

# Checklist:
